packages/x-charts/src/internals/index.ts

```diff
@@ -10,6 +10,7 @@ export { useInteractionItemProps } from '../hooks/useInteractionItemProps';
 export { useDrawingArea } from '../hooks/useDrawingArea';
 export { useScatterChartProps } from '../ScatterChart/useScatterChartProps';
 export { useScatterPlotData } from '../ScatterChart/useScatterPlotData';
+export { seriesConfig as scatterSeriesConfig } from '../ScatterChart/seriesConfig';
```

Comment:
Why is this necessary? 🤔

Reply:
I suppose because of the z scale value?

Reply:
Yes, I need to access `seriesConfig.colorProcessor`.